e2e_tests/tests/run/test_api.py

```diff
@@ -91,3 +91,78 @@ def test_run_kill_filter() -> None:
     for res in killResp.results:
         assert res.error == ""
         wait_for_run_state(sess, res.id, bindings.trialv1State.CANCELED)
+
+
+@pytest.mark.e2e_cpu
+def test_run_delete() -> None:
+    sess = api_utils.user_session()
+    exp_id = exp.create_experiment(
+        sess, conf.fixtures_path("no_op/empty_model_dir.yaml"), conf.fixtures_path("no_op")
+    )
+
+    searchResp = bindings.get_SearchRuns(
+        sess,
+        limit=1,
+        filter="""{"filterGroup":{"children":[{"columnName":"experimentId","kind":"field",
+        "location":"LOCATION_TYPE_RUN","operator":"=","type":"COLUMN_TYPE_NUMBER","value":"""
+        + str(exp_id)
+        + """}],"conjunction":"and","kind":"group"},"showArchived":false}""",
+    )
+
+    run_id = searchResp.runs[0].id
```

Comment:
nit: assert searchResp has a 0th index? rather than an confusing exception in tests.

Reply:
Moved tests outside of e2e for deletion since we no longer are doing/testing the kill action